samples/ServiceFabricRemoting/WeatherService.NetFx461/PackageRoot/ServiceManifest.xml

```diff
@@ -1,22 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ServiceManifest Name="WeatherService"
+<ServiceManifest Name="WeatherService_NetFx461"
                  Version="1.0.0"
                  xmlns="http://schemas.microsoft.com/2011/01/fabric"
                  xmlns:xsd="http://www.w3.org/2001/XMLSchema"
                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <ServiceTypes>
     <!-- This is the name of your ServiceType.
          This name must match the string used in the RegisterServiceAsync call in Program.cs. -->
-    <StatelessServiceType ServiceTypeName="WeatherService" />
+    <StatelessServiceType ServiceTypeName="WeatherService_NetFx461" />
   </ServiceTypes>
 
   <!-- Code package is your service executable. -->
   <CodePackage Name="Code" Version="1.0.0">
+    <!-- Install Datadog .NET Tracer -->
+    <SetupEntryPoint>
+      <ExeHost>
+        <Program>DatadogInstall.bat</Program>
+        <WorkingFolder>CodePackage</WorkingFolder>
+      </ExeHost>
+    </SetupEntryPoint>
     <EntryPoint>
       <ExeHost>
-        <Program>WeatherService.exe</Program>
+        <Program>WeatherService.NetFx461.exe</Program>
       </ExeHost>
     </EntryPoint>
+    <EnvironmentVariables>
+      <!-- Enable Datadog .NET Tracer -->
+      <EnvironmentVariable Name="CORECLR_ENABLE_PROFILING" Value="1" />
```

Comment:
Is this where we would recommend setting DD variables? Or in the `Config/Settings.xml`?

Reply:
Good question. AFAIK, you define env vars at the service level here (which you can override in `ApplicationManifest.xml`). If you use parameters in `Config/Settings.xml`, you would still need to pull them in as env vars by using a line like (where `WebApp_ASPNETCORE_ENVIRONMENT` is the parameter name):
https://github.com/DataDog/dd-trace-dotnet/blob/58d99c2080e91a973e0fcb602de224187af64511/samples/ServiceFabricRemoting/ServiceFabricApplication/ApplicationPackageRoot/ApplicationManifest.xml#L19